### PR TITLE
Changed 'id' to 'snapshotId'

### PR DESCRIPTION
**Pull request by gbggrant**

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/mongo/AgoraMongoDao.scala [hunk 1]
```diff
@@ -15,7 +15,7 @@ import org.broadinstitute.dsde.agora.server.model.AgoraEntity
 import spray.json._
 
 object AgoraMongoDao {
-  val MongoDbIdField = "_id"
+  val MongoDbIdField = "_snapshotId"
```
Comment (dshiga) [inline src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/mongo/AgoraMongoDao.scala]: Do we want to keep this one as _id? We really just want the field name in the methods entity to be called snapshotId. Here we're changing a field name in the counter collection.

Comment (gbggrant) [inline src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/mongo/AgoraMongoDao.scala]: Thanks for catching that.

**Comment by dshiga** [comment]
It looks good to me, but it might be good to get one more opinion.

**Comment by jacarey** [comment]
+1

**Pull request merged**